src/Avalonia.Controls/TopLevel.cs

```diff
@@ -917,7 +917,7 @@ void PlatformImpl_LostFocus()
             while (focused.VisualParent != null)
                 focused = focused.VisualParent;
 
-            if (focused == this)
```

Comment:
@maxkatz6 found another small issue with popups and input elements when the window is deactivated.